components/uid.rst

```diff
@@ -344,6 +344,14 @@ entity primary keys::
         // ...
     }
 
+.. caution::
+
+    Using UUIDs as primary keys is usually not recommended for performance reasons:
+    indexes are slower and take more space (because UUIDs in binary format take 128 bits
+    instead of 32/64 bits for auto-incremental integers) and the non-sequential nature of
+    UUIDs fragments indexes. UUID v7 is the only variant that solves the fragmentation
```

Comment:
WDYT to add anchor link on UUUD v7 ?

Reply:
Good idea! I did this while merging. Thanks.

Reply:
UUIDv6 are also fine from a fragmentation PoV

Reply:
I've updated the docs here: https://github.com/symfony/symfony-docs/commit/2cbc55e95efc57d0204684843732ddfee3507f06 Thanks!